swagger_server/controllers_local/controller.py

```diff
@@ -0,0 +1,524 @@
+#
+# /businesses
+#
+
+from flask import make_response, jsonify
+
+from swagger_server.configuration import ons_env
+from swagger_server.controllers_local import validate
+from swagger_server.controllers_local.util import filter_falsey_values, model_to_dict
+from swagger_server.models_local.model import Business, Party, Respondent, BusinessRespondent, Address
+
+db = ons_env
+
+
+#
+# /businesses
+#
+def businesses_get(searchString=None, skip=None, limit=None):
+    """
+    searches Businesses
+    By passing in the appropriate options, you can search for available Businesses
+    :param searchString: pass an optional search string for looking up Businesses
+    :type searchString: str
+    :param skip: number of records to skip for pagination
+    :type skip: int
+    :param limit: maximum number of records to return
+    :type limit: int
+
+    :rtype: None
+    """
+    return "to be implemented"  # pragma: no cover  # pragma: no cover
+
+
+def businesses_post(party):
+    """
+    adds a reporting unit of type Business
+    Adds a new Business, or updates an existing Business based on the business reference provided
+    :param party: Business to add
+    :type party: dict | bytes
+
+    :rtype: None
+    """
+
+    v = validate.Validator(validate.Exists('id', 'reference'), validate.IsUuid('id'))
```

Comment:
Like the pattern, just feel this line is difficult to read and doing too much, maybe split it up into its parts e.g. not have a function inside a function. Also if we could change one of these names it would help my brain, as effectively we are doing validate.Validator(..).validate